nodeadm/internal/kubelet/config.go

```diff
@@ -201,9 +201,11 @@ func (ksc *kubeletSubConfig) withNodeIp(cfg *api.NodeConfig, kubeletArguments ma
 
 func (ksc *kubeletSubConfig) withVersionToggles(kubeletVersion string, kubeletArguments map[string]string) {
 	// TODO: remove when 1.26 is EOL
-	// --container-runtime flag is gone in 1.27+
 	if semver.Compare(kubeletVersion, "v1.27.0") < 0 {
+		// --container-runtime flag is gone in 1.27+
 		kubeletArguments["container-runtime"] = "remote"
+		// --container-runtime-endpoint moved to kubelet config start from 1.27
+		kubeletArguments["container-runtime-endpoint"] = "unix:///run/containerd/containerd.sock"
```

Comment:
wow i didn't think to [check that](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md?plain=1#L1800-L1801), nice catch

Reply:
lemme update this into the comment!